build/tasks/test-on-node.js

```diff
@@ -59,24 +59,24 @@ module.exports = function( grunt ) {
 		var runDone = false;
 		var testActive = false;
 
-		QUnit.begin( function() {
+		QUnit.on( "runStart", function() {
 			grunt.log.ok( "Testing " + file + " ..." );
 		} );
-		QUnit.testStart( function() {
+		QUnit.on( "testStart", function() {
 			testActive = true;
 		} );
-		QUnit.log( function( details ) {
+		QUnit.on( "assert", function( details ) {
```

Comment:
This is kinda interesting. js-reporters doesn't define an `assert` event. To render individual assertions in progress, its necessary.

I think its fine for QUnit to add another event, but the object should follow the spec for `Assertion`.